torch/_inductor/lowering.py

```diff
@@ -1692,10 +1709,8 @@ def apply_constraint(arg, fx_arg):
     "torchvision::roi_align",
 }
 make_fallback(aten._adaptive_avg_pool2d_backward, require_dense)
-make_fallback(aten.convolution_backward, constrain_to_fx_strides)
 make_fallback(aten._cudnn_rnn, require_dense)
 make_fallback(aten._cudnn_rnn_backward, require_contiguous)
-make_fallback(aten.cumsum, require_dense, warn=False)
```

Comment:
cc @eellison just wanted to check if this is OK. It's needed to get the tests to pass

Reply:
Is there a way to look at the cpp decompositions ?

What does `aten.cumprod`, and `aten.sort` get mapped to now ? 

We need to replicate the existing `constrain_to_fx_strides` for convolution_backward but maybe that's happening organically and it's appearing as   as `aten.convolution`?

Reply:
@eellison these "new" decomps are all just coming from the original C++ CompositeImplicitAutograd decomps from the dispatcher. I can try to find a few of them so we can check.

Alternatively: what do you think of updating the fallback logic to "allow" a fallback, if we detect that the op in question has a CompositeImplicitAutograd registration? (so we'd be saying "actual no, inductor shouldn't just use CompositeImplicitAutograd decomps over fallbacks by default")

Reply:
> Alternatively: what do you think of updating the fallback logic to "allow" a fallback, if we detect that the op in question has a CompositeImplicitAutograd registration? (so we'd be saying "actual no, inductor shouldn't just use CompositeImplicitAutograd decomps over fallbacks by default")

This feels like less work though, compared to auditing the fallbacks in inductor and replacing the overload packet fallbacks with more finegrained fallbacks for overloads. So if this sounds reasonable to you, I'll probably just do this